sentry-core/src/main/java/io/sentry/core/UncaughtExceptionHandlerIntegration.java

```diff
@@ -19,7 +19,7 @@
   private boolean isRegistered = false;
   private UncaughtExceptionHandler threadAdapter;
 
-  public UncaughtExceptionHandlerIntegration() {
+  UncaughtExceptionHandlerIntegration() {
```

Comment:
Same as above. Now there's no public constructor. Is that what we want?

Reply:
@bruno-garcia right now it's a default integration and could be only removed as it's added on `init()`.
I would see one removing from `SentryOptions.getIntegrations()`, for that you won't need a ctor.

Reply:
Right, so we don't want users of this package to be able to use this integration by themselves. Outside of `Sentry.init`. 
Happy with that. It's always easier to open it later as opposed to restricting access.